config/keri/inception-config.json

```diff
@@ -10,4 +10,4 @@
   "ncount": 1,
   "isith": "1",
   "nsith": "1"
```

Comment:
Having a transferable Autonomic Identifier is great. We should implement the rotation mechanism feature later.